web/src/components/Footer.tsx

```diff
@@ -0,0 +1,87 @@
+import { Icon } from "@/components/njwds/Icon";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { MediaQueries } from "@/lib/PageSizes";
+import { useMediaQuery } from "@mui/material";
+import { ReactElement } from "react";
+
+export const Footer = (): ReactElement => {
+  const isLargeScreen = useMediaQuery(MediaQueries.desktopAndUp);
```

Comment:
[sand] Is there a way to ditch this media query hook in a follow-up refactor? It seems weird to me that we can't draw the footer without knowing the user's viewport width instead of just letting CSS reflow the content.

Reply:
I've noticed they don't work well with Flex / and when your manually changing the size of the view port

Reply:
let's chat in thread